src/components/MySpace/MySpace.stories.tsx

```diff
@@ -1,109 +1,100 @@
 import React from 'react'
 import { Meta } from '@storybook/react'
 import { ObjectId } from 'bson'
+import { testUser1 } from '../../__fixtures__/authUsers'
 import MySpace from './MySpace'
 import { mockRssFeedTwo } from '__mocks__/news-rss'
-import { GetMySpaceDocument } from 'operations/portal/queries/getMySpace.g'
 import { SPACEFORCE_NEWS_RSS_URL } from 'constants/index'
+import { AuthContext } from 'stores/authContext'
+import { Collection } from 'types'
 
 // Load 2 items
 const RSS_URL = `${SPACEFORCE_NEWS_RSS_URL}&max=2`
 
-export default {
-  title: 'Layouts/My Space',
-  component: MySpace,
-  decorators: [
-    (Story) => (
-      <div className="sfds">
-        <Story />
-      </div>
-    ),
-  ],
-  parameters: {
-    mockData: [
-      {
-        url: RSS_URL,
-        method: 'GET',
-        status: 200,
-        response: () => {
-          return mockRssFeedTwo
-        },
-      },
-    ],
-  },
-} as Meta
+const mockNewsWidget: any = {
+  _id: new ObjectId(),
+  title: 'Recent News',
+  type: 'News',
+}
 
-const exampleMySpaceData = [
+const exampleMySpaceData: Collection[] = [
   {
-    __typename: 'Collection',
-    _id: new ObjectId().toHexString(),
+    _id: new ObjectId(),
     title: 'Example Collection',
     type: 'Collection',
     bookmarks: [
       {
-        _id: new ObjectId().toHexString(),
+        _id: new ObjectId(),
         url: 'https://google.com',
         label: 'Webmail',
         cmsId: 'a',
         isRemoved: false,
       },
       {
-        _id: new ObjectId().toHexString(),
+        _id: new ObjectId(),
         url: 'https://mypay.dfas.mil/#/',
         label: 'MyPay',
         cmsId: 'b',
         isRemoved: false,
       },
       {
-        _id: new ObjectId().toHexString(),
+        _id: new ObjectId(),
         url: 'https://afpcsecure.us.af.mil/PKI/MainMenu1.aspx',
         label: 'vMPF',
-        cmsId: null,
-        isRemoved: false,
       },
     ],
   },
-  {
-    __typename: 'NewsWidget',
-    _id: new ObjectId().toHexString(),
-    title: 'Recent News',
-    type: 'News',
-  },
+  mockNewsWidget,
 ]
 
-export const ExampleMySpace = () => <MySpace bookmarks={[]} />
+const examplePortalUser = {
+  userId: 'BERNADETTE.CAMPBELL.5244446289@testusers.cce.af.mil',
+  mySpace: exampleMySpaceData,
+  displayName: 'BERNIE',
+  theme: 'light',
+}
 
-ExampleMySpace.parameters = {
-  apolloClient: {
-    mocks: [
-      {
-        request: {
-          query: GetMySpaceDocument,
-        },
-        result: {
-          data: {
-            mySpace: exampleMySpaceData,
-          },
-        },
-      },
-    ],
+const mockContext = {
+  user: testUser1,
+  portalUser: examplePortalUser,
+  setUser: () => {
+    return
+  },
+  setPortalUser: () => {
+    return
+  },
+  logout: () => {
+    return
+  },
+  login: () => {
+    return
   },
 }
 
-export const Loading = () => <MySpace bookmarks={[]} />
-
-Loading.parameters = {
-  apolloClient: {
-    mocks: [
```

Comment:
The user's mySpace now comes from a query in the `DefaultLayout` where a loading check is done. Removed this storybook component because it is now unnecessary.